skill-tree /src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT"/>
+    </root>
+    <logger name="org.testcontainers" level="INFO"/>
+    <logger name="com.github.dockerjava" level="WARN"/>
+    <logger name="com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.wire" level="OFF"/>
```

Comment:
These loggers I've used from the Medium blog I followed, the first one is suggesting to only print the `WARN` messages in case of docker logs and the second one is to not log the messages of Docker Java client to Apache Http Client.

Reply:
for the second one (it prints the logs of docker connecting to application) we can change the logging level to WARN if not INFO, https://stackoverflow.com/questions/77081390/exclude-from-logs-in-java-debug-com-github-dockerjava-zerodep-shaded-org-apach when its set to DEBUG it generates a lots of logs